web-platform/website/src/components/pages/login/LoginButton.tsx

```diff
@@ -40,5 +40,9 @@ const styles = {
     paddingBottom: Spacing.Medium,
     borderWidth: 2,
     fontWeight: 700,
-  }),
+  },
+  createResponsiveStyle(ScreenSize.ExtraSmall, {
+    padding: "10px"
```

Comment:
I've been avoiding custom values. Can we use Spacing enums that are close to this value?